docs/pages/guides/upgrading/1.0.mdx

```diff
@@ -0,0 +1,169 @@
+---
+meta:
+  title: "Upgrading to 1.0"
+  parentTitle: "Guides"
+  description: "Guide to upgrade to Liveblocks version 1.0"
+---
+
+```bash
+npm update @liveblocks/client@1.0 @liveblocks/node@1.0
+```
+
+The major release of 1.0 marks the maturity of Liveblocks. We added additional
+features to [Webhooks](/docs/guides/webhooks), enhancements to the `Room` class,
+and more. The primary change you should note is the transition to making the
+`userId` mandatory in the `authorize` option. This change is in line with our
+[new pricing model](https://liveblocks.io/pricing).
+
+Let’s take a look at the changes below!
+
+## Changes in @liveblocks/client [#client]
+
+### Adds throttle option to createClient [#throttle]
+
+You can now specify the `throttle` option in
+[`createClient`](https://liveblocks.io/docs/api-reference/liveblocks-client#createClientThrottle)
+that can go as low as 16ms. If not set, the default value is 100ms.
+
+```ts highlight="5"
+import { createClient } from "@liveblocks/client";
+
+const client = createClient({
+  /* ... other options ... */
+  throttle: 16,
+});
+```
+
+### Improvements to the Room class [#room]
+
+We added new methods to the `Room` class that you can use to obtain the storage
+status of a room, subscribe to storage status changes, or reconnect to a room
+whenever necessary. You can also use the
+[Liveblocks DevTools extension](/devtools) to visualize real-time changes to
+storage while implementing these changes.
+
+#### Room.getStorageStatus [#getStorageStatus]
+
+The different storage statuses are:
+
+- `not-loaded`: Initial state when entering the room.
+- `loading`: Once the storage has been requested via room.getStorage().
+- `synchronizing`: When some local updates have not been acknowledged by
+  Liveblocks servers.
+- `synchronized`: Storage is in sync with Liveblocks servers.
+
+#### Room.subscribe [#subscribe]
+
+Use `Room.subscribe` to subscribe to storage status changes. This method returns
+an unsubscribe function.
+
+```ts
+room.subscribe("storage-status", (status) => {
+  // Implement your logic here
+  switch (status) {
+    case "not-loaded":
+      break;
+    case "loading":
+      break;
+    case "synchronizing":
+      break;
+    case "synchronized":
+      break;
+    default:
+      break;
+  }
+});
+```
+
+#### Room.reconnect [#reconnect]
+
+You can use `room.reconnect()` to close the room connection and try to
+[reconnect](/docs/api-reference/liveblocks-client#Room.reconnect).
+
+## Changes in @liveblocks/node [#node]
+
+### Update the authorize option [#authorize]
+
+We have updated the `authorize` method to make `userId` mandatory. This change
+ties into our new [pricing model](https://liveblocks.io/pricing), which is based
+on Monthly Active Users (MAU) instead of connections. We use `userId` to track
+MAU associated with a Liveblocks account.
+
+If you do not use the `authorize` function, a cookie will be set on the client
```

Comment:
I would specify:

> If you do not use the `authorize` function but call the authorize with secret key API endpoint directly, you should pass the `userId` even if it's not mandatory on the API level. (Because it's used for the MAU etc....)

> If you do not use the `authorize` function because you are using the public API key method, a cookie will etc...

`Liveblocks/node` is a SDK with util methods to use our REST API. Currently, there is only a method for the authorize endpoint but we plan on adding all endpoints.

Some customers can still use the secret API key authentication method, but by calling the endpoint directly.